File: oim-server-master/oim-server-web/src/zone/general/kernel/work/module/group/main/entity/GroupMember.ts
/**
 * <br>
 * Date 2020-11-08 18:55:40<br>
 * @author XiaHui
 * @since 1.0.0
 */
export default class GroupMember {

    /**
     * 主键id
     */
    public id: string = '';
    /**
     * 创建时间（格式：2019-01-01 00:00:00）
     */
    public createdDateTime: string = '';
    /**
     * 创建时间戳（毫秒）
     */
    public createdTimestamp: string = '';
    /**
     * 是否逻辑删除：0：否、1：是
     */
    public isDeleted: number = 0;
    /**
     * 修改时间（格式：2019-01-01 00:00:00）
     */
    public updatedDateTime: string = '';
    /**
     * 修改时间戳（毫秒）
     */
    public updatedTimestamp: string = '';
    /**
     * 群id
     */
    public groupId: string = '';
    /**
     * 备注名(群中显示昵称)
     */
    public nickname: string = '';
    /**
     * 权限  1:群主 2:管理员 3:普通成员
     */
    public position: string = '';
    /**
     * 群成员用户id
     */
    public userId: string = '';
}

File: oim-fx-master/oim-work-starter/src/main/java/com/oimchat/client/general/kernel/work/module/group/entity/GroupMember.java
package com.oimchat.client.general.kernel.work.module.group.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

import com.onlyxiahui.app.basic.storage.bean.PrimaryStringEntity;


/**
 * 群成员 <br>
 * Date 2019-07-27 16:22:59<br>
 * 
 * @author XiaHui [onlovexiahui@qq.com]<br>
 * @since 1.0.0
 */
@Entity()
@Table(name = "w_group_member")
public class GroupMember extends PrimaryStringEntity {

	/**
	 * 群id
	 */


	@Column(nullable = false, length = 40)
	private String groupId;

	/**
	 * 群成员用户id
	 */


	@Column(nullable = false, length = 40)
	private String userId;

	/**
	 * 权限 1:群主 2:管理员 3:普通成员
	 */


	@Column(nullable = false, length = 2)
	private String position;

	/**
	 * 备注名(群中显示昵称)
	 */


	@Column(nullable = false, length = 100)
	private String nickname;

	public static final String position_owner = "1";// 群主
	public static final String position_admin = "2";// 管理员
	public static final String position_normal = "3";// 普通成员

	public String getGroupId() {
		return groupId;
	}

	public void setGroupId(String groupId) {
		this.groupId = groupId;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public String getPosition() {
		return position;
	}

	public void setPosition(String position) {
		this.position = position;
	}

	public String getNickname() {
		return nickname;
	}

	public void setNickname(String nickname) {
		this.nickname = nickname;
	}
}
